packages/firestore/src/local/simple_db.ts

```diff
@@ -63,14 +70,21 @@ export class SimpleDb {
           'Database "' + name + '" requires upgrade from version:',
           event.oldVersion
         );
-        // TODO(mikelehen): If/when we need to do an actual data
-        // migration, we'll want to wrap db in a SimpleDb and have the
-        // runUpgrade function return a PersistencePromise, since we'll
-        // likely need to do async reads and writes. For now we're
-        // cheating and just passing the raw IndexedDB in, since
-        // createObjectStore(), etc. are synchronous.
+        // We are provided a version upgrade transaction from the request, so
+        // we wrap that in a SimpleDbTransaction to allow use of our friendlier
+        // API for schema migration operations.
         const db = (event.target as IDBOpenDBRequest).result;
```

Comment:
Can you do the wrapping right here? That way, your comment is immediately related to the next line.

Reply:
Done.